sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue

```diff
@@ -0,0 +1,271 @@
+<template>
```

Comment:
Is this the component wich will be shared with the student? Should it be moved to somewhere common?